core/templates/components/card/card.twig

```diff
@@ -0,0 +1,72 @@
+{#
+/**
+ * @file
+ * Card Component.
+ *
+ * Basic building block component with two variants
+ *   - Horizontal: An image beside text content
+ *   - Vertical (default): An image above text content
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ * -- su-card--horizontal
+ * - card_image: A full HTML entity for an image.
+ * - card_super_headline: Text heading
+ * - card_headline: Text heading
+ * - card_body: An open variable for anything. Generally this would be for
+ *     WYSIWYG editor contents.
+ * - card_cta_link: The href value for the cta field link
+ * - card_cta_attributes: Any html attributes on the cta link that need to be
+ *     added. For example: _target.
+ * - card_cta_label: The clickable text for the CTA link
+ * - card_button_link: The href property of the button component.
+ * - card_button_label: The text label of the button component.
+ */
+#}
+<div{{ attributes }} class="su-card {{ modifier_class }}">
+
+  {# Card Image #}
+  {% if card_image is not empty %}
+    {{ card_image }}
+  {% endif %}
+
+  <div class="su-card__contents">
+
+  {# CTA Super Headline #}
```

Comment:
In these comments, you probably want to replace 'CTA' with 'Card'

Reply:
This has been changed.